apps/frontend/src/components/global/upload_tabs/FileReader.vue

```diff
@@ -17,7 +17,7 @@
         <v-col cols="12" align="center">
           <div class="d-flex flex-column justify-center">
             <span :class="title_class">Heimdall</span>
-            <span :class="title_class">Lite</span>
+            <span v-if="serverMode" :class="title_class">Lite</span>
```

Comment:
This is currently displaying backwards, needs to be `v-if="!serverMode"`, currently displays 'lite' on heimdall server and 'heimdall' on heimdall lite.